alphagov/paas-cf
# [#159136450] Use Bosh DNS

What
----

We have enabled Bosh DNS using the 1.8 release. This is an essential precursor to enabling service discovery. Service discovery is not enabled yet, this only changes the DNS mechanism from Consul to Bosh DNS. Consul has not been removed as part of this work - this will be played in a subsequent story.

Since we'll be using bosh-dns in the future and we only have monitoring for consul we've also added a simple process check to check that the bosh-dns process is running on all of the machines that we used to check the consul agent on.

How to review
-------------

1. Code review
1. Follow the steps in https://github.com/alphagov/paas-bootstrap/pull/178
1. Deploy to a dev environment with DATADOG_ENABLED=true
1. Check that the `$yourEnv Bosh-dns process running` monitor is in datadog 🐶 

Who can review
--------------

Not @LeePorte or @richardTowers
